docker/Dockerfile

```diff
@@ -1,3 +1,7 @@
 FROM python:3.10-slim-bullseye
 ARG VERSION
+RUN apt-get update && \
```

Comment:
Can we add a comment for why we need this?

Reply:
added!